cmd/tiller-proxy/main.go

```diff
@@ -80,14 +81,19 @@ func main() {
 
 	config, err := rest.InClusterConfig()
 	if err != nil {
-		log.Fatalf("Unable to get cluter config: %v", err)
+		log.Fatalf("Unable to get cluster config: %v", err)
 	}
 
 	kubeClient, err = kubernetes.NewForConfig(config)
 	if err != nil {
 		log.Fatalf("Unable to create a kubernetes client: %v", err)
 	}
 
+	appRepoClient, err := appRepo.NewForConfig(config)
+	if err != nil {
+		log.Fatalf("Unable to create an app repository client: %v", err)
```

Comment:
Is the `NewForConfig` method trying to create a new `AppRepository` CRD? From the `tiller-proxy-rbac.yml`, it seems the proxy can only `get` the resources.

Reply:
Nope, it's a standard method which all CRDs and the core k8s api provide, to create a client for interacting with the server. The autogenerated functions enabled by the client are in `pkg/client/clientset/versioned/typed/apprepository/v1alpha1/apprepository.go` (including `Create`) :)

Reply:
Thanks for the explanation!